nuxt.config.ts

```diff
@@ -59,40 +91,13 @@ export default defineNuxtConfig({
   // https://v3.nuxtjs.org/api/configuration/nuxt.config#ignore
   i18n: {
     defaultLocale: 'de',
-    locales: [
-      {
-        code: 'de',
-        name: 'Deutsch',
-      },
-      {
-        code: 'en',
-        name: 'English',
-      },
-      {
-        code: 'es',
-        name: 'Español',
-      },
-      {
-        code: 'fr',
-        name: 'Français',
-      },
-      {
-        code: 'ja',
-        name: '日本語',
-      },
-    ],
+    locales: supportedLocales,
     vueI18n: {
       fallbackLocale: 'en',
       legacy: false,
       locale: 'de',
       // TODO: can this be cleaner via some sort of iteration? (not sure)
```

Comment:
I think it's fine to keep `localeMessages` as-is tbh. IMO it's easier to read

Reply:
Actually ya, i initially did this to try and explicitly import for the languageSelector but i ended up not needing to. reverting.

Reply:
Nvm just double checked and realized it is still needed and being used for the tests. I think it's fine as is